docker/debian-buster-amd64-openssl-1.1.x/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM node:lts-buster-slim
+
+WORKDIR /usr/src/app
+COPY --from=app . ./
+COPY --from=utils . ./
+
+# Update system dependencies.
+# OpenSSL is missing in the base image, so we install it
+RUN apt-get update -y \
+  && apt-get install -y openssl
+
+# expected target architecture, retrieved by `uname -m`
+ARG EXPECTED_UNAME_ARCH="x86_64"
+ENV EXPECTED_UNAME_ARCH=$EXPECTED_UNAME_ARCH
+RUN ./uname.sh
+
+# Only OpenSSL 1.1 is expected to be on this system
+RUN ldconfig -p | grep ssl
```

```suggestion
RUN ldconfig -p | grep ssl | sed "s/.*=>\s*//"
```
Missing or intentional?